src/drt/src/dr/FlexDR_maze.cpp

```diff
@@ -2622,6 +2623,18 @@ void FlexDRWorker::processPathSeg(frMIdx startX,
                   z,
                   i - 1 >= 0 ? &points[i - 1] : nullptr,
                   i + 2 < (int) points.size() ? &points[i + 2] : nullptr);
+  } else {
+    if (layer->isHorizontal() == vertical)
```

Comment:
I think something needs to get changed in the code because this line is super unintuitive. Maybe use an [`Orientation2D`](https://github.com/The-OpenROAD-Project/OpenROAD/blob/ba39bf1a326bf1d492e7ebe11049916247f9a6a5/src/odb/include/odb/isotropy.h#L98) instead of `bool vertical`?

Reply:
Orient2D is good.  At least
```
 if (layer->isVertical() != seg_is_vertical)  // wrong way seg
```
[rename vertical to seg_is_vertical]